wls-briefwahl-service/src/test/java/de/muenchen/oss/wahllokalsystem/briefwahlservice/configuration/UnicodeConfigurationTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c): it@M - Dienstleister für Informations- und Telekommunikationstechnik
+ * der Landeshauptstadt München, 2024
+ */
+package de.muenchen.oss.wahllokalsystem.briefwahlservice.configuration;
+
+import static de.muenchen.oss.wahllokalsystem.briefwahlservice.TestConstants.SPRING_NO_SECURITY_PROFILE;
+import static de.muenchen.oss.wahllokalsystem.briefwahlservice.TestConstants.SPRING_TEST_PROFILE;
+import static de.muenchen.oss.wahllokalsystem.briefwahlservice.TestConstants.TheEntityDto;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import de.muenchen.oss.wahllokalsystem.briefwahlservice.MicroServiceApplication;
+import de.muenchen.oss.wahllokalsystem.briefwahlservice.domain.TheEntity;
+import de.muenchen.oss.wahllokalsystem.briefwahlservice.rest.TheEntityRepository;
+import java.net.URI;
+import java.util.UUID;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.test.context.ActiveProfiles;
+
+@SpringBootTest(
+        classes = { MicroServiceApplication.class },
+        webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT,
+        properties = {
+                "spring.datasource.url=jdbc:h2:mem:testexample;DB_CLOSE_ON_EXIT=FALSE",
+                "refarch.gracefulshutdown.pre-wait-seconds=0"
+        }
+)
+@ActiveProfiles(profiles = { SPRING_TEST_PROFILE, SPRING_NO_SECURITY_PROFILE })
+class UnicodeConfigurationTest {
+
+    private static final String ENTITY_ENDPOINT_URL = "/theEntities";
+
+    /**
+     * Decomposed string: String "Ä-é" represented with unicode letters "A◌̈-e◌́"
+     */
+    private static final String TEXT_ATTRIBUTE_DECOMPOSED = "\u0041\u0308-\u0065\u0301";
+
+    /**
+     * Composed string: String "Ä-é" represented with unicode letters "Ä-é".
+     */
+    private static final String TEXT_ATTRIBUTE_COMPOSED = "\u00c4-\u00e9";
+
+    @Autowired
+    private TestRestTemplate testRestTemplate;
+
+    @Autowired
+    private TheEntityRepository theEntityRepository;
+
+    @Test
+    @Disabled
```

Comment:
Warum ist der eigentlich disabled? In meinem Generat steht das nicht, der Test schlägt halt fehl...